<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>media</class>
 <widget class="QMainWindow" name="media">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>media</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>49</x>
      <y>69</y>
      <width>521</width>
      <height>331</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_nome">
     <property name="geometry">
      <rect>
       <x>164</x>
       <y>46</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Nome</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_nome">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>44</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_nota1">
     <property name="geometry">
      <rect>
       <x>164</x>
       <y>90</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Nota 1</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_nota1">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>88</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_nota2">
     <property name="geometry">
      <rect>
       <x>164</x>
       <y>135</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Nota 2</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_nota2">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>130</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_nota3">
     <property name="geometry">
      <rect>
       <x>165</x>
       <y>179</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Nota 3</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_nota3">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>175</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_calcular">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>230</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>CÃ¡lculo</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
